volatility3/framework/constants/linux/__init__.py

```diff
@@ -347,3 +347,88 @@ def flags(self) -> str:
 MODULE_MAXIMUM_CORE_SIZE = 20000000
 MODULE_MAXIMUM_CORE_TEXT_SIZE = 20000000
 MODULE_MINIMUM_SIZE = 4096
+
+
+TAINT_FLAGS = {
```

Comment:
it would be better a dictionary of [dataclasses](https://docs.python.org/3/library/dataclasses.html)

Reply:
Done !

Reply:
It looks much cleaner and more polished now.